tests/test_query.py

```diff
@@ -258,6 +287,28 @@ def test_get_invalid_min_num_sessions(
     response.status_code = 422
 
 
+def test_get_valid_assessment(test_app, mock_successful_get, monkeypatch):
```

Comment:
Same here: nothing wrong with moving things around. But there should be a reason so we don't move it again next week. So in the future let's make an issue for this, discuss the idea of reordering, and then make a separate PR for the reordering so it doesn't get mixed in with unrelated functional changes.